src/util/hass-util.html

```diff
@@ -26,7 +26,7 @@
 window.hassUtil.DOMAINS_WITH_MORE_INFO = [
   'alarm_control_panel', 'automation', 'camera', 'climate', 'configurator',
   'cover', 'fan', 'group', 'history_graph', 'light', 'lock', 'media_player', 'script',
-  'sun', 'updater', 'vacuum', 'input_datetime',
+  'sun', 'updater', 'vacuum', 'input_datetime', 'plug',
```

Comment:
`'plug'` is not a home assistant domain.

Reply:
Thanks, I removed it from there.